pom.xml

```diff
@@ -31,9 +31,8 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.642.4</jenkins.version>
-    <java.level>7</java.level>
-    <jenkins-test-harness.version>2.30</jenkins-test-harness.version>
+    <jenkins.version>2.60.3</jenkins.version>
```

Comment:
Any particular reason you are pushing it this high? I have no problem with it, but you seemed to be saying that you felt compelled to do so, which I am not following.

Reply:
The PCT fails without this bump.  I can do more investigation into alternative fixes, but this was the only way that I could fix it without spending an undue amount of time working on this.